source/create.js

```diff
@@ -56,7 +57,7 @@ const create = defaults => {
 
 	Object.assign(got, {...errors, mergeOptions: merge.options});
 	Object.defineProperty(got, 'defaults', {
-		value: deepFreeze(defaults),
+		value: defaults.mutable ? defaults : deepFreeze(defaults),
 		writable: false,
```

Comment:
Should we set `writable` to true if mutation is enabled? I may have a set of options objects that I'd like to just swap out with an assignment.